### PR TITLE
Implement cityScape scopes

### PR DESCRIPTION
**Pull request by afjlambert**
This feature introduces the concept of 'scopes' for CityScapes (and monuments). A cityscape-scope can be of type municipal or national. This is now reflected in the graphql server. I had to deprecate to keep schema in sync for Monument + CityScape which both are of type Restriction. The type-field on monument is therefore deprecated.

The frontend is now using the scope to show the appropriate message to the user. We still do not show cityScape information for the IMTR-flow, but we _do_ for the OLO-flow.

### CODE REVIEW

File: apps/client/src/__mocks__/addressMockNoCityScape.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+export default {
```
Comment (robinpiets) [inline apps/client/src/__mocks__/addressMockNoCityScape.js]: Any good to convert this file to TS?

File: apps/client/src/__mocks__/addressMockNoMonument.js [hunk 2]
```diff
@@ -2,7 +2,13 @@ export default {
   houseNumberFull: "123",
```
Comment (robinpiets) [inline apps/client/src/__mocks__/addressMockNoMonument.js]: Any good to convert this file to TS?

File: apps/graphql/src/loaders/cityScape/cityScape.test.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
+const cityScape = require("rewire")(".");
+const { reducer } = cityScape.__get__("loader");
+
+describe("cityScape", () => {
+  test("reducer", async () => {
+    expect(reducer(require("./__mocks__/1477.json"))).toMatchSnapshot();
```
Comment (robinpiets) [inline apps/graphql/src/loaders/cityScape/cityScape.test.js:6]: snapshot testing? 🤞

**Review by svenjens — APPROVED** [review]

**Review by robinpiets — APPROVED** [review]

**Comment by robinpiets** [comment]
Please squash

**Pull request merged**